atompub.go

```diff
@@ -3,11 +3,12 @@ package esatomdatapg
 import (
 	"crypto/rand"
```

Comment:
Should probably group these imports.
https://github.com/golang/go/wiki/CodeReviewComments#imports